v3/src/components/case-card/case-card.v2.js

```diff
@@ -501,30 +532,51 @@ iDataContext.doSelectCases({
                   }
                 }.bind(this),
                 editAttribute = function () {
-                  this.updateAttribute = function(iAttrRef, iChangedAttrProps) {
-                    updateAttribute(iChangedAttrProps)
-                  }
-                  var attributePane = DG.AttributeEditorView.create({attrRef: {attribute: iAttr}, attrUpdater: this})
-                  attributePane.append()
+                  this.editAttributePropModal(iAttr.get('id'), true)
                 }.bind(this),
 
                 editFormula = function () {
-                  iContext.invokeLater(function () {
-                    DG.DataContextUtilities.editAttributeFormula(iContext, iCollection,
-                        iAttr.get('name'), iAttr.get('formula'))
-                  }, 30)
+                  this.editFormulaModal(iAttr.get('id'), true)
                 }.bind(this),
 
                 hideAttribute = function () {
-                  DG.DataContextUtilities.hideAttribute(iContext, iAttr.get('id'))
+                  const attrId = iAttr.get('id')
+                  iCaseMetadata?.applyModelChange(
+                    () => iCaseMetadata?.setIsHidden(attrId, true),
+                    {
+                      notifications: hideAttributeNotification([attrId], iContext.data),
+                      undoStringKey: "DG.Undo.caseTable.hideAttribute",
+                      redoStringKey: "DG.Redo.caseTable.hideAttribute"
+                    }
+                  )
```

Comment:
For some of these you could reduce the changes in this already large file by replacing the implementation of `DG.DataContextUtilities.hideAttribute` with the implementation here, and then we can eventually eliminate the unused portions of `DG.DataContextUtilities` so that what remains serves as an interface between v2 and v3.